cpp/test/Common/TestHelper.cpp

```diff
@@ -347,3 +340,30 @@ Test::TestHelper::shutdownOnInterrupt()
     _ctrlCHandler->setCallback(shutdownOnInterruptCallback);
 }
 #endif
+
+void
+Test::TestHelper::addDefaultPluginFactories(std::vector<Ice::PluginFactory>& factories) const
+{
+    // TODO: we should not register these plug-ins all the time.
```

Comment:
The difficulty is we no longer have an ICE_STATIC_LIBS or similar define.

It would be much better to check a runtime value.

Reply:
Fixed.